styles/colors-silicon-night.less

```diff
@@ -49,8 +49,9 @@
 
 // Checkbox
 // ---------------------------------------
-@agate-checkbox-text-color: @agate-white;
-@agate-checkbox-bg-color: @agate-accent;
+@agate-checkbox-text-color: @silicon-white;
```

Comment:
This line isn't necessary. The value is defined as white in the day version (which is inherited by this skin).